ui/guest/src/classes/ElementRegistry.ts

```diff
@@ -186,11 +199,38 @@ export function getRect(id: number): DOMRect {
   return get(id).element.getBoundingClientRect();
 }
 
+export function fromRecord(record: ElementRecord, iceId: number): RegistryEntry {
+  if (!record) {
+    return null;
+  }
+  const { id, element, label, modelId, index, fieldId: fieldIds } = record;
+  return { id, element, modelId, index, label, fieldId: fieldIds[0], iceId };
```

Comment:
This is fixed to always read the first field ID, disregarding the actual field ID on the desired ice record.

Reply:
yep, and that is ok, there are another method to read all the ID's